test_conformance/api/test_api_min_max.cpp

```diff
@@ -19,6 +19,12 @@
 #include <ctype.h>
 #include <string.h>
 
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
```

Comment:
Could we use `std::min` here vs. defining our own `MIN`?

Reply:
done, thanks very much.